services/index.js

```diff
@@ -8,9 +8,22 @@ const serviceRegex = /\/services\/(.*)\/\1\.js$/;
 
 function loadServiceClasses() {
   // New-style services
-  return glob.sync(`${__dirname}/**/*.js`)
+  const services = glob.sync(`${__dirname}/**/*.js`)
     .filter(path => serviceRegex.test(path))
-    .map(path => require(path))
+    .map(path => require(path));
+
+  const serviceClasses = [];
+  services.forEach(service => {
+    if (typeof service === 'function') {
+      serviceClasses.push(service);
+    } else {
+      for (const serviceClass in service) {
+        serviceClasses.push(service[serviceClass]);
+      }
+    }
+  });
+
+  return serviceClasses;
```

Comment:
I think it is sensible that we should change `loadServiceClasses()` so we can define a service file `gem.js` which exports multiple classes, like this:

```js
module.exports = {
  GemVersion,
  GemDownloads,
  GemOwner,
  GemRank,
}
```

instead of forcing one class per file, but I'm open to feedback on this point.